### PR TITLE
Curator auth

### LINKED ISSUE #2: Curator authentication
Curator can log in and see main page and admin panel

### PR DESCRIPTION
**Pull request by sparksofl**
Close #2
Feature: Curator authentication
Now curator can log in and see main page and admin panel.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,6 @@
 
 config/database.yml
 config/secrets.yml
+
+git-ssh
```
Comment (brytannia) [inline .gitignore]: do not add your key to gitignore

File: .gitignore [hunk 2]
```diff
@@ -21,3 +21,6 @@
...
+git-ssh.pub
```
Comment (brytannia) [inline .gitignore]: there always must be an endlline

File: app/admin/dashboard.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+ActiveAdmin.register_page "Dashboard" do
+
+  menu priority: 1, label: proc{ I18n.t("active_admin.dashboard") }
+
+  content title: proc{ I18n.t("active_admin.dashboard") } do
+    
+    columns do
+      column do
+        panel "Last Registered Users" do
```
Comment (brytannia) [inline app/admin/dashboard.rb:9]: Nice :+1:

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+
+  def configure_permitted_parameters
+    temp = [:first_name, :last_name, :email, :password, :password_confirmation]
+    temp.each { |p| devise_parameter_sanitizer.for(:sign_up) << p }
+  end
+
+  def authenticate_admin_user!
+    redirect_to new_user_session_path unless current_user.try(:is_admin?)
+  end
+end
```
Comment (brytannia) [inline app/controllers/application_controller.rb]: newline https://help.github.com/articles/dealing-with-line-endings/

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  # before_action :authenticate_admin_user!
```
Comment (brytannia) [inline app/controllers/application_controller.rb]: why you don't use this callback?

File: config/database.yml.example~ [hunk 6]
```diff
@@ -0,0 +1,85 @@
+# PostgreSQL. Versions 8.2 and up are supported.
```
Comment (brytannia) [inline config/database.yml.example~]: this file is a dublication of database.yml.example file. please, remove it

File: app/controllers/application_controller.rb (hunk 5 above)
Comment (sparksofl) [inline app/controllers/application_controller.rb]: 'This webpage has a redirect loop'. Even when

``` ruby
before_action :authenticate_admin_user!, except: [ :new, :create, :sign_in ]
```

Comment (brytannia) [inline app/controllers/application_controller.rb]: I've seen that :authenticate_admin_user! is used in active admin config. so here you don't need to use this method. so delete str.3. Let's stick to the rule that any commented code haven't be pushed to repo.

File: app/views/layouts/.fuse_hidden00002b640000001b [hunk 7]
```diff
@@ -0,0 +1,24 @@
+doctype html
```
Comment (brytannia) [inline app/views/layouts/.fuse_hidden00002b640000001b]: I'm not sure that this file have to be pushed to remote. I guess, it's some temporary file. please delete it and google why it's created.

File: app/views/layouts/.fuse_hidden00002b640000001c [hunk 8]
```diff
@@ -0,0 +1,24 @@
+doctype html
```
Comment (brytannia) [inline app/views/layouts/.fuse_hidden00002b640000001c]: the same as in previous case.

File: app/controllers/.fuse_hidden00002d9a0000001d [hunk 9]
```diff
@@ -0,0 +1,17 @@
+class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :authenticate_admin_user!, except: [ :new, :create, :sign_in ]
```
Comment (brytannia) [inline app/controllers/.fuse_hidden00002d9a0000001d:3]: this callback works properly?

Comment (sparksofl) [inline app/controllers/.fuse_hidden00002d9a0000001d:3]: Forgot to comment. Nope, it doesn't

Comment (brytannia) [inline app/controllers/.fuse_hidden00002d9a0000001d:3]: @sparksofl better to delete this string than commented it

File: app/controllers/.fuse_hidden00002d9a0000001d [hunk 10]
```diff
@@ -0,0 +1,17 @@
+class ApplicationController < ActionController::Base
```
Comment (brytannia) [inline app/controllers/.fuse_hidden00002d9a0000001d:1]: this file haven't been added to commit. I guess it's a some sublime issue.

File: app/views/layouts/.fuse_hidden00002b640000001b (hunk 7 above)
Comment (sparksofl) [inline app/views/layouts/.fuse_hidden00002b640000001b]: http://askubuntu.com/questions/493198/what-is-a-fuse-hidden-file-and-why-do-they-exist

Comment (brytannia) [inline app/views/layouts/.fuse_hidden00002b640000001b]: so let's delete them from this commit and after that - add them to gitignore

File: app/controllers/.fuse_hidden00002d9a0000001d (hunk 10 above)
Comment (sparksofl) [inline app/controllers/.fuse_hidden00002d9a0000001d:1]: Can't even pull that hidden file back. I've added it to .gitignore (just in case)

Comment (brytannia) [inline app/controllers/.fuse_hidden00002d9a0000001d:1]: it's fine for now. however, please, find out how to avoid this issue in future.

**Comment by brytannia** [comment]
:+1: Good job!

**Pull request merged**